dmutils/logging.py

```diff
@@ -85,10 +85,21 @@ def after_request(response):
         return response
 
     logging.getLogger().addHandler(logging.NullHandler())
+    handlers = [logging.StreamHandler(sys.stdout)]
```

Comment:
Do we think this should be configurable per-app, like the other handler?

Reply:
@katstevens can we leave as-is and if we find a use case to make it configurable per-app we could always add it later?

Reply:
Sounds reasonable 😸